video/out/w32_common.c

```diff
@@ -1682,6 +1697,8 @@ static LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam,
         // the window is created. Otherwise, fallback to setting alphanumeric mode on
         // the first keypress.
         SetTimer(w32->window, (UINT_PTR)WM_CREATE, 250, NULL);
+        w32->menu = CreatePopupMenu();
+        w32->menu_ctx = talloc_new(w32);
```

Comment:
I would prefer this to be `mp_win32_menu_init` and both menu and menu_ctx should be in opaque menu_ctx struct pointer, that is defined in menu.c. If we move impl to other file, let's move all the detail there.

Reply:
> Also then you don't have dummy talloc ctx, because your whole object is itself a context.

I think the dump talloc ctx is still necessary, although it is not an issue now. but if we want to add another child pointer in menu_ctx, the new pointer may be freed by the `talloc_free_children(menu_ctx*)` call in `mp_win32_menu_update`.